tests/test_forward.py

```diff
@@ -19,13 +21,13 @@ def callback(data):
 CALLBACKS = {ENDPT_NAME: callback}
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
 def farm():
     """Create a coco runner."""
     return endpoint_farm.Farm(N_HOSTS, CALLBACKS)
 
 
-@pytest.fixture
```

Comment:
why?

Reply:
I added a second test function that can use the same coco server to run. Setting the fixture scope to module just means it doesn't have to take down the previous server and start a new one. Saves only about 1s in the run time.